src/main/java/edu/ie3/datamodel/io/csv/DefaultInputHierarchy.java

```diff
@@ -0,0 +1,271 @@
+/*
+ * © 2020. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.datamodel.io.csv;
+
+import edu.ie3.datamodel.exceptions.FileException;
+import edu.ie3.datamodel.models.UniqueEntity;
+import edu.ie3.datamodel.models.input.MeasurementUnitInput;
+import edu.ie3.datamodel.models.input.NodeInput;
+import edu.ie3.datamodel.models.input.OperatorInput;
+import edu.ie3.datamodel.models.input.RandomLoadParameters;
+import edu.ie3.datamodel.models.input.connector.LineInput;
+import edu.ie3.datamodel.models.input.connector.SwitchInput;
+import edu.ie3.datamodel.models.input.connector.Transformer2WInput;
+import edu.ie3.datamodel.models.input.connector.Transformer3WInput;
+import edu.ie3.datamodel.models.input.connector.type.LineTypeInput;
+import edu.ie3.datamodel.models.input.connector.type.Transformer2WTypeInput;
+import edu.ie3.datamodel.models.input.connector.type.Transformer3WTypeInput;
+import edu.ie3.datamodel.models.input.graphics.GraphicInput;
+import edu.ie3.datamodel.models.input.system.*;
+import edu.ie3.datamodel.models.input.system.characteristic.EvCharacteristicInput;
+import edu.ie3.datamodel.models.input.system.characteristic.WecCharacteristicInput;
+import edu.ie3.datamodel.models.input.system.type.*;
+import edu.ie3.datamodel.models.input.thermal.ThermalUnitInput;
+import edu.ie3.datamodel.models.timeseries.TimeSeries;
+import edu.ie3.datamodel.models.timeseries.repetitive.LoadProfileInput;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.apache.commons.io.FilenameUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Default directory hierarchy for input models */
+public class DefaultInputHierarchy implements FileHierarchy {
+  private static final Logger logger = LoggerFactory.getLogger(DefaultInputHierarchy.class);
+
+  /** Use the unix file separator here. */
+  protected static final String FILE_SEPARATOR = "/";
```

Comment:
any reason why this is used here instead of `File.separator();`? Does this work within all OS?